light-client/src/consensus/rpc/portal_rpc.rs

```diff
@@ -0,0 +1,316 @@
+use crate::consensus::rpc::ConsensusRpc;
+use async_trait::async_trait;
+use ethportal_api::consensus::light_client::bootstrap::LightClientBootstrap;
+use ethportal_api::light_client::bootstrap::LightClientBootstrapCapella;
+use ethportal_api::light_client::finality_update::LightClientFinalityUpdateCapella;
+use ethportal_api::light_client::optimistic_update::LightClientOptimisticUpdateCapella;
+use ethportal_api::light_client::update::LightClientUpdateCapella;
+use ethportal_api::types::consensus::light_client::finality_update::LightClientFinalityUpdate;
+use ethportal_api::types::consensus::light_client::optimistic_update::LightClientOptimisticUpdate;
+use ethportal_api::types::consensus::light_client::update::LightClientUpdate;
+use ethportal_api::types::content_key::beacon::LightClientFinalityUpdateKey;
+use ethportal_api::types::content_key::beacon::LightClientOptimisticUpdateKey;
+use ethportal_api::{
+    BeaconContentKey, BeaconContentValue, ContentValue, LightClientBootstrapKey,
+    LightClientUpdatesByRangeKey,
+};
+use eyre::eyre;
```

Comment:
I'm guessing the usage of this library is since it's grandfathered in from the original code we forked? Would it be a heavy lift (in a future pr) to convert over to `anyhow` so we have consistency amongst our codebases?

Reply:
Yeah, I will create an issue after the merge.